src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/BOM/MaterialForm/MaterialForm.tsx

```diff
@@ -20,7 +20,7 @@ const schema = yup.object().shape({
   manufacturerName: yup.string().required('Select a Manufacturer'),
   manufacturerPartNumber: yup.string().required('Manufacturer Part Number is required!'),
   nerPartNumber: yup.string().optional(),
-  quantity: yup.number().required('Enter a quantity!'),
+  quantity: yup.number().integer().required('Enter a quantity!'),
```

Comment:
Is this always an integer? could something be bought in non-whole-number pounds or something

Reply:
very good question

Reply:
I think wee always intended it to just be a whole number, but let me check

Reply:
yeah good catch, it should be able to be a float, I'll make the change

Reply:
ok so that's gonna require a schema change, I'm gonna make those tickets for now, cause we need fodder for the new ppl